cmd/scollector/main.go

```diff
@@ -34,39 +36,91 @@ var (
 	flagFilter          = flag.String("f", "", "Filters collectors matching this term, multiple terms separated by comma. Works with all other arguments.")
 	flagList            = flag.Bool("l", false, "List available collectors.")
 	flagPrint           = flag.Bool("p", false, "Print to screen instead of sending to a host")
-	flagHost            = flag.String("h", "", `Bosun or OpenTSDB host. Ex: "http://bosun.example.com:8070".`)
-	flagColDir          = flag.String("c", "", `External collectors directory.`)
 	flagBatchSize       = flag.Int("b", 0, "OpenTSDB batch size. Used for debugging bad data.")
-	flagSNMP            = flag.String("s", "", "SNMP host to poll of the format: \"community@host[,community@host...]\".")
-	flagICMP            = flag.String("i", "", "ICMP host to ping of the format: \"host[,host...]\".")
-	flagVsphere         = flag.String("v", "", `vSphere host to poll of the format: "user:password@host[,user:password@host...]".`)
 	flagFake            = flag.Int("fake", 0, "Generates X fake data points on the test.fake metric per second.")
 	flagDebug           = flag.Bool("d", false, "Enables debug output.")
-	flagFullHost        = flag.Bool("u", false, `Enables full hostnames: doesn't truncate to first ".".`)
-	flagTags            = flag.String("t", "", `Tags to add to every datapoint in the format dc=ny,rack=3. If a collector specifies the same tag key, this one will be overwritten. The host tag is not supported.`)
 	flagDisableMetadata = flag.Bool("m", false, "Disable sending of metadata.")
 	flagVersion         = flag.Bool("version", false, "Prints the version and exits.")
-	flagDisableDefault  = flag.Bool("n", false, "Disable sending of scollector self metrics.")
-	flagHostname        = flag.String("hostname", "", "If set, use as value of host tag instead of system hostname.")
-	flagFreq            = flag.String("freq", "15", "Set the default frequency in seconds for most collectors.")
 	flagConf            = flag.String("conf", "", "Location of configuration file. Defaults to scollector.conf in directory of the scollector executable.")
-	flagAWS             = flag.String("aws", "", `AWS keys and region, format: "access_key:secret_key@region".`)
+	flagToToml          = flag.String("totoml", "", "Location of destination toml file to convert. Reads from value of -conf.")
 
 	mains []func()
 )
 
-func readConf() {
+type Conf struct {
+	// Host is the OpenTSDB or Bosun host to send data.
+	Host string
+	// FullHost enables full hostnames: doesn't truncate to first ".".
+	FullHost bool
+	ColDir   string
```

Comment:
Need documentation on everything here. Whatever docs were on the flag should be ok.